README.md

```diff
@@ -0,0 +1,52 @@
+# Concordium Swift SDK
+
+**STATUS: EARLY DEVELOPMENT**
+
+An SDK for applications written in the [Swift Programming Language](https://www.swift.org/) to
+interact with the Concordium Blockchain.
+
+The main purpose of the SDK is to facilitate development of mobile wallet apps for iOS devices.
+
+### Supported platforms
+
+- iOS 15+
```

Comment:
Should we add a comment in Package.swift about updating this section of the README in case the supported platforms or versions change?

Reply:
done